### ISSUE TITLE
Add Python 3 support

### ISSUE THREAD

**Issue by icemac** [zopefoundation/z3c.etestbrowser#6]
Currently this package is not even importable because it needs the `ConfigParser` module in `setup.py`:

```
Getting distribution for 'z3c.etestbrowser==2.0.1'.
Traceback (most recent call last):
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 157, in save_modules
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 198, in setup_context
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 255, in run_setup
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 285, in run
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 253, in runner
  File ".../setuptools-33.1.1-py3.6.egg/setuptools/sandbox.py", line 47, in _execfile
  File "/var/folders/hn/hb5zqzw16lz93c_5ztm0vkj00000gn/T/easy_install-ottvutu3/z3c.etestbrowser-2.0.1/setup.py", line 17, in <module>
    description="Multi user address book application",
ModuleNotFoundError: No module named 'ConfigParser'
```

**Comment by mgedmin** [comment]
z3c.etestbrowser is a think wrapper around zope.testbrowser.

Good news: zope.testbrowser supports Python 3.

Bad news: zope.testbrowser had to shed large chunks of its API to do that.  z3c.etestbrowser will likely need to do the same.

**Comment by icemac** [comment]
@mgedmin I am using `z3c.etestbrowser.wsgi.Browser` together with the latest `zope.testbrowser` release without any problems. There is already #4 where I am not sure if it is necessary. So I hope porting this package to Python 3 would not be too hard.

**Comment by icemac** [comment]
There is already an old py3 branch: https://github.com/zopefoundation/z3c.etestbrowser/compare/py3 maybe something can be used from there.

**Comment by mgedmin** [comment]
I've memory like a goldfish: you did mention the less-than version pin in the PR description, with a future resolution in sight.  I'm pretty sure I read that before I switched to the diff.

**Comment by mgedmin** [comment]
And now I'm commenting in the wrong browser tab. *sigh*

**Comment by icemac** [comment]
Released in 3.0.
